spec/unit/messages/users_list_message_spec.rb

```diff
@@ -84,10 +99,32 @@ module VCAP::CloudController
         expect(message).to be_valid
       end
 
+      it 'accepts partial_usernames and origins' do
```

Comment:
The ` it 'accepts usernames without origins'` is now (correctly) part of the new context/describe `describe 'usernames_or_partial_usernames'` and can be removed from this context/describe.

Reply:
thanks for pointing this out!